recipes/kealib/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.4.6" %}
+
+package:
+    name: kealib
+    version: {{ version }}
+
+source:
+    fn: kealib-{{ version }}.tar.gz
+    url: https://bitbucket.org/chchrsc/kealib/downloads/kealib-{{ version }}.tar.gz
+    md5: 789174bd519736ac1e726613b6eb7672
+
+build:
+    number: 0
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
+        
+requirements:
+    build:
+        - python  # [win]    
+        - cmake
+        - hdf5 1.8.15.1
+
+    run:
+        - hdf5 1.8.15.1
```

Comment:
Thanks for clarification  :smiley:
What is the problem with netcdf? Does it not support the latest hdf5?